packs/data/pathfinder-bestiary-3.db/wyrmwraith.json

```diff
@@ -1339,7 +1557,7 @@
                     "value": 2
                 },
                 "description": {
-                    "value": "<p>The wyrmwraith unleashes a burst of negative energy that deals [[/r {18d6}[negative]]]{18d6 negative damage} to all creatures in a @Template[type:cone|distance:60] (@Check[type:reflex|dc:38|basic:true] save). They can't use Breath Weapon again for [[/br 1d4 #rounds]]{1d4 rounds}.</p>"
+                    "value": "<p>The wyrmwraith unleashes a burst of negative energy that deals [[/r {18d6}[negative]]]{18d6 negative damage} to all creatures in a @Template[type:cone|distance:60] (@Check[type:reflex|dc:38|basic:true] save).</p>\n<p>They can't use Breath Weapon again for [[/br 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>"
```

Comment:
Is the line break in the PDF? Reviewing from my laptop.

Reply:
I remember we had that discussion about adding additional line breaks for better readability some while ago and committed use this. I think for most of he recharge abilities there is a line break for the text containing the recharge.

Reply:
I've been removing them as I catch them. I don't remember discussing adding line breaks in paragraphs. I think we settled on adding line breaks when it came to Trigger, Effect, Requirements, etc. but not things in the text itself which we normally have been formatting according to how Paizo does it.